dependent/Turcica.csl

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Revue TURCICA </title>
+    <title-short>TURCICA </title-short>
+    <id>http://www.zotero.org/styles/revue-turcica</id>
+    <link rel="self" href="http://www.zotero.org/styles/revue-turcica"/>
```

Comment:
Please add add a link to the template style you used using `rel="template"`
Please add a link to the citation guidelines using `rel="documentation"`